### PR TITLE
Post new ad

### PR DESCRIPTION
**Pull request by empafrontend**
## Describe your changes
* adjustments and styling for post new ad (not 100% done)

## Issue ticket number and link
* issue #11 (https://github.com/empafrontend/Rentium/issues/11)

## Checklist before requesting a review

- [x] Issue fixed
- [x] Code compiles correctly
- [ ] Extended the README / documentation, if necessary

- [ ] Created tests which fail without the change (if Cypress tests have set up)
- [ ] All tests passing (if Cypress tests have set up)

### CODE REVIEW

File: src/NewAdPage.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    .matches(
+      /((https?):\/\/)?(www.)?[a-z0-9]+(\.[a-z]{2,}){1,3}(#?\/?[a-zA-Z0-9#]+)*\/?(\?[a-zA-Z0-9-_]+=[a-zA-Z0-9-%]+&?)?$/,
+      'Enter correct url'
+    )
+    .required('Please enter website'),
+  description: yup
+    .string()
+    .required('Required')
+    .min(10, 'Username must be between 10 to 500 characters')
+    .max(500, 'Username must be between 10 to 500 characters'),
```
Comment (millie-wy) [inline src/NewAdPage.tsx:26]: "username" should be "description"

**Review by millie-wy — CHANGES_REQUESTED** [review]
Looks good, just found a place where the text is not updated. 
New issues for the missing input fields, the textfield with various rows and function for form submission should be added too 😺

File: src/NewAdPage.tsx (hunk 1 above)
Comment (empafrontend) [reply src/NewAdPage.tsx:26]: i fix! :)

**Review by millie-wy — APPROVED** [review]
Looks all fine 🎄

**Review by millie-wy — APPROVED** [review]

**Pull request merged**